source/administration/config-settings.rst

```diff
@@ -3793,6 +3793,14 @@ There are a number of settings customizable in ``config.json`` unavailable in th
 Service Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Data Prefetch
+^^^^^^^^^^^^^^
+When enabled, messages in all unread channels are fetched from the server whenever the client reconnects to the network to eliminate loading time when users switch to unread channels.
```

Comment:
If it's true and false, we normally list them separately explaining what happens when it's true and when it's false.